package.json

```diff
@@ -16,7 +16,7 @@
     "release": "cd dist && npm publish",
     "lint": "tslint 'src/**/*.ts'",
     "preversion": "yarn build && yarn test:dist",
-    "coverage": "jest src --coverage"
+    "coverage": "jest src --coverage --coverageReporters text text-summary"
```

Comment:
Sorry, can we add this to the `jest.config.js` instead?

```
  "coverageReporters": ['text', 'text-summary'],
  "coverageThreshold": {
    "global": { lines: 50 }
  },
```

Id imagine coverage is pretty abysmal atm though